### ISSUE TITLE
resign success but can't install on my phone

### ISSUE THREAD

**Issue by liushengxu** [dabear/iphoneipa-resign#2]
Hi ,I try many times regisn my ipa with "developer mobileprovision" and "Distribution Mobileprovison" 
but it seems doesn't work。 it can't install on my phone  .

is it really work ? or what should I pay attention to ?

waiting for your reply ,Thanks!

**Comment by dabear** [comment]
If you're resigning isoxdripreader, check that your mobileprovisionfile contains healthkit entitlements (can be verified in the apple dev portal web site).

Other than that I don't know.

**Comment by liushengxu** [comment]
yes ， i trying resign isoxdripreader  and the  healthkit entitlements  and APNS is contains .  I will look for any other reasons.
thanks 
![image](https://user-images.githubusercontent.com/7800566/28773563-ab4ef21a-761c-11e7-8230-60fd00b3cd1b.png)

**Comment by dabear** [comment]
seems the sign time did not change..
